openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-limitsmonitor/src/tools/LimitsMonitor/LimitsControl.vue

```diff
@@ -508,8 +509,8 @@ export default {
       }
     },
     update() {
-      if (this.screenItems.length !== 0) {
-        this.api.get_tlm_values(this.screenItems).then((data) => {
+      if (this.availableItems.length !== 0) {
+        this.api.get_tlm_values(this.availableItems).then((data) => {
           this.updateValues(data)
```

Comment:
Is the calculation of available just to optimize this `update()` method so it doesn't call `updateValues()` if there are no available items?

Reply:
available is required once we implement the time series database.  Just getting ahead on that.